examples/python/route_guide/run_codegen.py

```diff
@@ -28,5 +27,16 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-# Runs the protoc with gRPC plugin to generate protocol messages and gRPC stubs.
-python -m grpc.tools.protoc -I../../protos --python_out=. --grpc_python_out=. ../../protos/route_guide.proto
```

Comment:
Recapping an offline conversation:  I'm more accepting of this change because we are committing to having protoc.main as part of a GA API.

I'm still not in love with this as a public API though, it seems awkward to have a python function that takes a series of shell style argument strings.  If we are going to vend a way to generate protos in python code, I would much rather see explicit arguments for each property.